adding container to quiz game

ibrahimaljarad opened the pull request:
this is just a container t quiz game with the test file,

ibrahimaljarad committed:
adding contanier to quiz game

tmansuroglu commented on src/containers/QuizGamePage/index.jsx:
```diff
@@ -0,0 +1,14 @@
+const QuizGamePage = () => {
+  return (
+    <>
+      <Navbar />
```
Navbar will be visible without Navbar component here.

tmansuroglu replied:
I will add a fix for it

ibrahimaljarad replied:
PTAL

tmansuroglu reviewed (approved):
Nice work!

ibrahimaljarad committed:
quiz game screenshot added

tmansuroglu commented on src/locales/en.json:
```diff
@@ -93,6 +93,12 @@
     }
   },
   "games": {
+    "quizgame":{
+      "screenshot":{
+        "title":"Quiz Game",
+        "src":"https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
+      }
+    },
```
"quizgame":{
      "title":"Quiz Game",
      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7",
      ...
    }

I was referring to something like this

ibrahimaljarad replied:
PTAL

tmansuroglu replied:
nice

ibrahimaljarad committed:
fixed the json file

ibrahimaljarad committed:
run fix

ibrahimaljarad committed:
fixed the conflict

ibrahimaljarad committed:
fixed the conflict

ibrahimaljarad committed:
fixed the test

majdajroudi commented on src/locales/en.json:
```diff
@@ -96,6 +96,10 @@
   },
   "games": {
+    "quizgame": {
+      "title": "Quiz Game",
```
I think we should also be adding these to the ar.json and tr.json files

ibrahimaljarad replied:
PTAL

ibrahimaljarad committed:
adding turkish and arabic

louisrli commented on src/App.jsx:
```diff
@@ -25,6 +26,7 @@ function App() {
         <Route path="/login" component={""} />
         <Route path="/games/memorygame" component={MemoryGame} />
         <Route exact path="/games/puzzle" component={PuzzlePage} />
+        <Route path="/games/quizgame" component={QuizGamePage} />
```
@tmansuroglu can you please rename your component to MemoryGamePage. you can see that there is an inconsistency here in the naming

tmansuroglu replied:
sure

louisrli commented on src/locales/ar.json:
```diff
@@ -96,6 +96,11 @@
   "games": {
+    "quizgame": {
+      "title": "لعبة الاسئلة",
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```
this doesn't have to be in the json. unless its different per translation

just declare a variable `const QUIZ_GAME_IMAGE = <the url>` and use it in code

louisrli replied:
also you added these translations but where are they used?

ibrahimaljarad replied:
actually i added this image and title so tarkan can use them in the game page, we will have a game page has the 3 games with this screenshot of each game.

ibrahimaljarad replied:
removed the screenshot

louisrli commented on src/locales/en.json:
```diff
@@ -96,6 +96,10 @@
   "games": {
+    "quizgame": {
+      "title": "Quiz Game",
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```
notice how the 3 are duplicated = already a warning

ibrahimaljarad replied:
object removed

ibrahimaljarad committed:
removing the screenshot

ibrahimaljarad committed:
run fix

ibrahimaljarad committed:
deleting quizgame object

louisrli reviewed (approved):
so you removed the translations completely? pr looks fine as it is though

louisrli committed:
Merge branch 'dev' into quizGameContainer

ibrahimaljarad commented:
> so you removed the translations completely? pr looks fine as it is though

i just removed the screenshot object which we added it as extra for the games page

ibrahimaljarad commented:
@Ammar-64  can you merge please

louisrli commented:
this PR used to have a translations file and now it doesnt

louisrli commented:
that translations file also had a translated title

louisrli reviewed (changes requested):
my understanding was you were gonna delete only the line that had the screenshot

ibrahimaljarad commented:
> my understanding was you were gonna delete only the line that had the screenshot

the previous object with the translation has nothing to do with this pr, this pr is only for making a container for the quiz game so when tarkan asked for only a title and a screenshot for adding it to the general games page i added it here,so it has nothing to do with the translation

louisrli commented:
and that's how it should be, a clean separation of PRs in that case :)

Ammar-64 committed:
Merge branch 'dev' into quizGameContainer

Ammar-64 commented:
@ibrahimaljarad the branch has a conflict, can you fix it please?

ibrahimaljarad commented:
@Ammar-64  can we merge now please,

Ammar-64 committed:
Merge branch 'dev' into quizGameContainer
